graylog2-server/src/main/java/org/graylog2/rest/resources/system/SessionsResource.java

```diff
@@ -188,7 +188,7 @@ public SessionValidationResponse validateSession(@Context ContainerRequestContex
                 session.setAttribute(HTTPHeaderAuthenticationRealm.SESSION_AUTH_HEADER, usernameHeader.get());
             }
 
-            LOG.debug("Session created {}", session.getId());
+            LOG.debug("Session created");
```

Comment:
how about logging the user here instead?

Reply:
Done, thanks!